views/partials/header.hbs

```diff
@@ -1,13 +1,11 @@
 <div class="header">
-  <a href="/"><h1 class="text-center naz-events">Nazareth Events</h1></a>
-  <a href="/add-event" class="glyphicon glyphicon-plus"></a>
+  <a href="/add-event"><i class="fa fa-plus	plus" alt="add event"></i>
```

Comment:
No need to have a separate `i` tag really, just put the class on the `a` tag instead